js/server/tests/aql/aql-functions-string.js

```diff
@@ -37,8 +37,7 @@ var assertQueryError = helper.assertQueryError;
 var assertQueryWarningAndNull = helper.assertQueryWarningAndNull;
 
 const wrapFunctions = [
```

Comment:
This was basically added for the NOOPT(V8( and should be removed completely again.